recipes/open62541/all/test_v1_package/FooFlt.NodeSet2.xml

```diff
@@ -0,0 +1,282 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
remove all these files please, they are in test_package so it can be reused

Reply:
resolved in 1dc45894e95cb5d5a87d867699bb3aa5f1a9582d